File: app/src/main/java/social/entourage/android/invite/view/InvitationsAdapter.java
package social.entourage.android.invite.view;

import social.entourage.android.api.model.TimestampedObject;
import social.entourage.android.base.EntourageBaseAdapter;
import social.entourage.android.map.tour.information.discussion.ViewHolderFactory;

/**
 * Created by mihaiionescu on 08/08/16.
 */
public class InvitationsAdapter extends EntourageBaseAdapter {

    public InvitationsAdapter() {

        ViewHolderFactory.registerViewHolder(
                TimestampedObject.INVITATION_CARD,
                new ViewHolderFactory.ViewHolderType(InvitationCardViewHolder.class, InvitationCardViewHolder.getLayoutResource())
        );

    }

}

File: app/src/main/java/social/entourage/android/invite/view/InvitationsAdapter.kt
package social.entourage.android.invite.view

import social.entourage.android.api.model.TimestampedObject
import social.entourage.android.base.EntourageBaseAdapter
import social.entourage.android.base.ViewHolderFactory.ViewHolderType
import social.entourage.android.invite.view.InvitationCardViewHolder.Companion.layoutResource

/**
 * Created by mihaiionescu on 08/08/16.
 */
class InvitationsAdapter : EntourageBaseAdapter() {
    init {
        viewHolderFactory.registerViewHolder(
                TimestampedObject.INVITATION_CARD,
                ViewHolderType(InvitationCardViewHolder::class.java, layoutResource)
        )
    }
}